app/index.tsx

```diff
@@ -7,9 +7,29 @@ import { LedgerBitcoinWallet } from './hooks/useLedgerBitcoinWallet';
 import { LedgerClient } from './ledgerIpc';
 import { ipcRenderer } from 'electron';
 import { createBrowserHistory } from 'history';
+import * as Sentry from '@sentry/react';
+import { captureException, showReportDialog } from '@sentry/react';
 
+Sentry.init({
+  dsn:
+    'https://c4bbd637d7154e338433faf792b7c320@o461938.ingest.sentry.io/5471443'
+});
 const AppContainer = process.env.PLAIN_HMR ? Fragment : ReactHotAppContainer;
 
+ipcRenderer.on('USER_ERROR_REPORT', () => {
+  const error = new Error('Submit user feedback');
+  captureException(error);
+  showReportDialog({
+    title: 'Something went wrong? Tell us.',
+    subtitle:
+      'If you need instant help please fill in a proper name and email!',
```

Comment:
We do not need a "proper name" to help them.
```suggestion
      'If you need help please provide your email',
```